scripts/build.js

```diff
@@ -5,6 +5,11 @@ const btoa = require("btoa")
 
 // login credentials
 const { PERSONAL_ACCESS_TOKEN, USERNAME } = process.env
+if (PERSONAL_ACCESS_TOKEN === undefined) {
```

Comment:
wdyt about adding a check for `USERNAME` and using `!PERSONAL_ACCESS_TOKEN` for consistency across the codebase?

Reply:
Oops. Careless. Fixed